.tekton/wait-for-image-task.yaml

```diff
@@ -30,9 +34,12 @@ spec:
 
       echo "Image $(params.IMAGE) found."
 
-      digest="$(skopeo inspect \
+      infos="$(skopeo inspect \
         --retry-times 10 \
-        --format '{{.Digest}}' \
+        --format '{{.Digest}} {{ index .Labels "vcs-ref" }} {{ index .Labels "source-location" }}' \
```

Comment:
As I mentioned in my other comment, this task should be the place to check the presence of `vcs-ref` and `source-location` labels and to show error when they aren't set.

Reply:
This will also mean that `source-location` has to be kept in all `konflux*Dockerfile`-s, not just the Scanner-V2 and Collector ones.

https://github.com/stackrox/stackrox/pull/13577#discussion_r1893601603